### DISCUSSION TITLE
Cython support?

### DISCUSSION THREAD

**Discussion by ArtemIsmagilov** [zed-industries/zed#14407] [Feature Requests]
Do you have Cython syntax support?

**Comment by ArtemIsmagilov** [comment]
Are you planning Cython support? My files and syntax were not recognized.

**Reply by SomeoneToIgnore** [reply]
Zed team does not plan anything related to languages at the moment.
There's https://github.com/zed-industries/extensions and a way to add a new set of tree-sitter grammars for a certain language + optionally, LSP server support on top.

So either that path should be taken for Cyton support, or some improvement for Python support should be done in https://github.com/zed-industries/zed/tree/main/crates/languages/src/python

**Comment by lgeiger** [comment]
@ArtemIsmagilov I published [Cython language extension for Zed](https://github.com/lgeiger/zed-cython) based on [`tree-sitter-cython`](https://github.com/b0o/tree-sitter-cython). For now it provides syntax highlighting and outline support and can be installed via the Zed extensions UI.

**Reply by ArtemIsmagilov** [reply]
Thanks!
